### ISSUE TITLE
Cannot fetch from the XKCD API

### ISSUE THREAD

**Issue by wang-yi-yao** [wang-yi-yao/wang-yi-yao.github.io#3]
Fetch is blocked by CORS policy, not sure how to fix this.

**Comment by wang-yi-yao** [comment]
The error shown on the browser:
![Screenshot 2021-05-24 155702](https://user-images.githubusercontent.com/42727264/119295801-d42bc580-bcab-11eb-9688-930de8969020.png)

Current code:
```
<!--StartFragment-->
try {        
 gettingData = fetch("http://xkcd.com/info.0.json/", {        
  method: "GET",			
  headers: {                
   "Mode":"no-cors",				
   "Accept": "application/json",                
   "Content-Type":"application/json"			
  }        
 });   
} catch(error) {        
 console.log("Something went wrong, please check for errors");    
}

<!--EndFragment-->
```

**Comment by wang-yi-yao** [comment]
I might look into using Node.Js as a solution. Obviously changing the API settings is out of my reach, so the other fix is use a middleware some people have described to fetch the JSON from the server to browser. If I can get Node.Js to work as a middleware, then the rest should be smooth sailing.

**Comment by wang-yi-yao** [comment]
I stopped developing this particular experiment for my website. It got out of scope extremely quickly. On further research, GitHub pages is intended to host static websites which takes in only HTML, CSS and JS. I will return to experimenting with public APIs in the future. But for now it is better if I just complete this website and utilising resources from my own repo. It is a shame that the CORS policy stopped this from being fully developed.

**Comment by wang-yi-yao** [comment]
I stopped developing this particular experiment for my website. It got out of scope extremely quickly. On further research, GitHub pages is intended to host static websites which takes in only HTML, CSS and JS. I will return to experimenting with public APIs in the future. But for now it is better if I just complete this website and utilising resources from my own repo. It is a shame that the CORS policy stopped this from being fully developed.
